release-content/0.16/release-notes/occlusion_culling.md

```diff
@@ -0,0 +1,32 @@
+When it comes to pushing hardware to support larger levels and more detailed meshes, "simply draw less stuff" is a tried-and-true strategy.
+These techniques are collectively referred to as "culling": we can save work whenever we can determine "this doesn't need to be drawn" for cheaper than it would cost to draw the thing despite its pixels later being overwritten completely.
+
+**Occlusion culling** is the idea that we don't need to draw something that's completely blocked by other opaque objects,
+from the perspective of the camera.
+That makes sense! We don't need to draw a car parked on the other side of a building (even if it's within the range used for fustrum culling), and we *never* need to draw all of the teeth inside of a character's closed mouth.
+
+Before you worry too much: we're already checking for this category of wasted work in some form, via the use of a **depth prepass**.
+A depth prepass renders a simplified version of the scene, recording only the depth (distance from the camera) of each object into the 2-dimensional depth buffer.
+We can check the resulting depth buffer, count how many pixels are associated with each object, and then discard objects whose pixels were completely overwritten.
+As a result, we can avoid most fragment shading costs (dominated by textures and lighting) for occluded objects, but the vertex shading overhead is still present, in addition to the inherent cost of this fragment testing process.
+
+We can do better than that.
+As [PR #17413] lays out, we're adopting the modern two-phase occlusion culling (in contrast to a traditional potentially visible sets design) already used by our virtual geometry rendering, because it works well with the GPU-driven rendering architecture (cold specialization, retained bins) that we've established during this cycle!
+
+For now, this feature is marked as experimental, due to known precision issues that can mark meshes as occluded even when they're not.
```

Comment:
Maybe mention that this could result in pop in. It might be easier to understand the tradeoff for a lot of people.

Reply:
It's not really pop in, it'll be meshes not rendering when they're supposed to.

Reply:
Sure, but that will just last a few frames and it will eventually get rendered again which will cause pop in right?

Reply:
Not entirely sure, but don't think so. This is about over-agressive culling. If you shift the camera it might render again, though.